tool_requirements.py

```diff
@@ -11,4 +11,8 @@
         'min_version': '2020.03-SP2',
         'as_needed': True
     }
```

Comment:
I think you might have missed a comma here

Reply:
Absolutely, thanks for noting that. I was already debugging the documentation build but had no clue what was wrong ;-)